### ISSUE TITLE
Project dir argument not used when running debug

### ISSUE THREAD

**Issue by gobbledygook88** [fishtown-analytics/dbt#1733]
### Describe the bug
When running `dbt debug --project-dir=<some directory>` in another directory to that of your dbt project, the `dbt_project.yml` is not used.

### Steps To Reproduce
With an existing dbt project:

```bash
$ cd /tmp
$ dbt debug --project-dir=<path/to/project> --profiles-dir=$HOME/.dbt/
```

we get the following output:

```
Using profiles.yml file at /app/src/pipeline/profiles.yml

Configuration:
  profiles.yml file [OK found and valid]
  dbt_project.yml file [ERROR not found]
  profile: some-profile-name [OK found]
  target: production [OK found]
```

### Expected behavior
Would expect the output to be the following:

```
Using profiles.yml file at /app/src/pipeline/profiles.yml

Configuration:
  profiles.yml file [OK found and valid]
  dbt_project.yml file [OK found and valid]
  profile: some-profile-name [OK found]
  target: production [OK found]
```

### System information
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [x] bigquery
- [ ] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
installed version: 0.14.0
   latest version: 0.14.1
```

Also tried with version 0.14.1. and obtained the same behavior.

**The operating system you're using:**
Ubuntu 19.04

**The output of `python --version`:**
Python 3.7.4

### Additional context
After a brief debugging session, we found the `get_nearest_project_dir()` function in `dbt/core/dbt/task/base.py`. This function is called when running `dbt compile ` and `dbt run`, but not `dbt debug`.

Inspecting the parsed arguments shows the `project_dir` attribute is set correctly.

When running `dbt debug` in the actual project directory, dbt is able to pick up `dbt_project.yml` correctly.

**Comment by drewbanin** [comment]
Thanks for the report @gobbledygook88! Your diagnosis sounds spot-on to me

**Comment by beckjake** [comment]
Fixed in #1989
